src/app/modules/main/wallet_section/send/view.nim

```diff
@@ -242,7 +242,7 @@ QtObject:
     self.suggestedRoutesReady(newQVariant(self.transactionRoutes))
 
   proc suggestedRoutes*(self: View, amount: string) {.slot.} =
-    self.delegate.suggestedRoutes(self.selectedSenderAccount.address(), self.selectedRecipient,
+    self.delegate.suggestedRoutes("", self.selectedSenderAccount.address(), self.selectedRecipient,
```

Comment:
why dont we set it here as well? anyways causes conflicts with send modal flow, can be useful for sendModal also